en/docs/index.md

```diff
@@ -26,8 +30,7 @@ template: templates/single-column.html
                     </div>
                     <div class="card-content">
                         <p class="title">Security Processes and Programs</p>
-                        <a href="http://www.google.com"></a>
-                        <p class="hint">Read our security processes and programs to understand how we manage various aspects of security in our products.</p>
+                        <p class="hint">How we manage various aspects of security in WSO2 products</p>
```

Comment:
```suggestion
                        <p class="hint">Learn how we manage various aspects of security in WSO2 products</p>
```

Reply:
Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8